ping-thing-client.mjs

```diff
@@ -100,15 +91,13 @@ async function pingThing() {
     }
     try {
       try {
+        const latestBlockhash = await getLatestBlockhash();
```

Comment:
Pull the latest blockhash out of the read-through cache here and jam it into the transaction message.